src/main/java/pl/piekoszek/gorskimatches/equation/EquationMathChecker.java

```diff
@@ -2,11 +2,36 @@
 
 import org.springframework.stereotype.Component;
 
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
 @Component
 class EquationMathChecker {
 
+    Map<String, Boolean> allEquations = new HashMap<>();
+    ArrayList equations = new ArrayList();
+
+    {
+        for (int i = 0; i < 10; i++) {
+            for (int j = 0; j < 10; j++) {
+                for (int k = 0; k < 10; k++) {
+                    equations.add(i + "+" + j + "=" + k);
+                    equations.add(i + "-" + j + "=" + k);
+                    for (int l = 0; l < equations.size(); l++) {
+                        allEquations.put((String) equations.get(l), isMathematicallyCorrect((String) equations.get(l)));
+                        allEquations.remove(equations.get(l), false);
+
+                    }
+                }
+            }
+        }
+        Set<String> keySet = allEquations.keySet();
+        ArrayList<String> listOfMathCorrEquations = new ArrayList<String>(keySet);
```

Comment:
Should be written as `List<String> listOfMathCorrEquations = new ArrayList<>(keySet);`

Reply:
also why it's `mathCorr` instead of `mathCorrect` or `mathematicallyCorrect`  ?